src/include/daos_errno.h

```diff
@@ -92,7 +92,9 @@ extern "C" {
 	/** Incompatible durable format version */			\
 	ACTION(DER_DF_INCOMPT,		(DER_ERR_DAOS_BASE + 23))	\
 	/** Record size error */					\
-	ACTION(DER_REC_SIZE,		(DER_ERR_DAOS_BASE + 24))
+	ACTION(DER_REC_SIZE,		(DER_ERR_DAOS_BASE + 24))	\
+	/** XXX: Use it only when needs to restart the DTX */		\
+	ACTION(DER_RESTART,		(DER_ERR_DAOS_BASE + 25))
```

Comment:
We talked about DER_TX_RESTART in the call to avoid people thinking this is a generic error which would also make the comment clarification less necessary